okhttp3/pom.xml

```diff
@@ -31,6 +31,8 @@
     <module.name>zipkin2.reporter.okhttp3</module.name>
 
     <main.basedir>${project.basedir}/..</main.basedir>
+    <!-- pre-1.8, pre-kotlin, and last version before Call.timeout. -->
```

Comment:
from brave, as is most infrastructure in this PR, as brave does exactly the same thing for instrumentation. https://github.com/openzipkin/brave/blob/master/instrumentation/okhttp3/pom.xml#L34